cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -210,13 +211,55 @@
                 {{ notification('warning', true,
                     'Sorry, there were no results based on your filter selections.',
                     'Please reset the filter or change your selections and try again.') }}
+            {% elif posts.paginator.count == 1 %}
```

Comment:
Doesn't look like I can use `pluralize` in a concatenation scenario like this. I get a `TemplateSyntaxError: expected token ',', got ':'`.